so_vector/README.md

```diff
@@ -0,0 +1,49 @@
+## StackOverflow vector track
+
+This track benchmarks advanced searches on dense vector fields. Unlike the
+`dense_vector` track, each document contains other fields besides vectors to
+support testing features like ANN with filtering and hybrid search.
+
+The dataset is derived from this dump of StackOverflow posts: <TODO add link here>
+
+It only contains question documents -- all documents representing answers have
+been removed. Each question title was encoded into a vector using the sentence
+transformer model
+[multi-qa-mpnet-base-cos-v1](https://huggingface.co/sentence-transformers/multi-qa-mpnet-base-cos-v1).
+
+Each question and answer have formatted into a JSON document with the following fields:
```

Comment:
@gregoireco I documented here how we would like the documents to look. Instead of `embedding_title` I used `titleVector` to match the other field names. I also removed some fields that are not useful.

We could just make sure to use these names in the script. No worries about regenerating the large dataset from scratch, we can always transform it later to tweak these fields (avoiding the truly expensive step of embedding the text into a vector).